config/detekt/detekt.yml

```diff
@@ -180,130 +180,22 @@ exceptions:
     active: true
     excludes: "**/test/**,**/androidTest/**,**/*.Test.kt,**/*.Spec.kt,**/*.Spek.kt"
     exceptionNames:
-     - ArrayIndexOutOfBoundsException
-     - Error
-     - Exception
-     - IllegalMonitorStateException
-     - NullPointerException
-     - IndexOutOfBoundsException
-     - RuntimeException
-     - Throwable
+      - ArrayIndexOutOfBoundsException
+      - Error
+      - Exception
+      - IllegalMonitorStateException
+      - NullPointerException
+      - IndexOutOfBoundsException
+      - RuntimeException
+      - Throwable
     allowedExceptionNameRegex: "^(_|(ignore|expected).*)"
   TooGenericExceptionThrown:
     active: true
     exceptionNames:
-     - Error
-     - Exception
-     - Throwable
-     - RuntimeException
-
-formatting:
-  active: true
-  android: false
-  autoCorrect: true
-  AnnotationOnSeparateLine:
-    active: false
-    autoCorrect: true
```

Comment:
Why are we removing this? We should keep the default settings.

Reply:
@gouri-panda I hardly see 

> @MohitMaliFtechiz Can we remove the unnecessary code formatting's which is unrelated? This will also stable code coverages.

I hardly see how code formatting should change the overall codecoverage. Or do you mean something else?

Reply:
@kelson42  We are adding a lot of lines by just code formatting it. From codecoverage's perspective each function should coverage the test cases. I think that's why we are seeing drop in code coverage.

Reply:
> Why are we removing this? We should keep the default settings.

I have updated detekt in new version formatting is by default we don't need this code.

Reply:
hi @gouri-panda  latest detekt and klint requires all the  code formatting , lots of code formatting rules  are introduced on latest code so i did all the formatting , let me fix code coverage issue. this is coming mainly from two files , i am checking over it.

Reply:
If both of you agree that new code formatting is OK, then we should keep it. I don't like when code is not formatted properly. Side question: We don't have a code formating checker in our CI?

Reply:
@kelson42 ,

Earlier code formatting checker klint was outdated so its was checking code formatting as per their version , now since its has to updated along with gradle so new formatting rules has to applied. So I was getting lots of issue by upgrading it so that why i change many files.

Reply:
@MohitMaliFtechiz In which workflow is the code formatting checker?

Reply:
@kelson42 We are checking this static analysis job in pull request workflows

Reply:
> Maybe we deleted the default settings that's why it created new one. We always code formatted before merge it other wise our static analysis fails. @kelson42 yes, we have 2 library that does this one is lint which let's us check kotlin format and the other one is Detekt which analysis our code plus it can check our coding format.We can see this in our static analysis.

Hi @gouri-panda ,

please refer to https://github.com/detekt/detekt/pull/4352/commits/0603c9a3e27359536a4a27845226bd476e62396c they have removed formatting in update and make it as default property.

Reply:
That's great! Sorry! I haven't updated with this. But is our configuration pure default?